core/src/edu/asu/c3simulator/screens/BusinessDirectonScreen.java

```diff
@@ -0,0 +1,159 @@
+	 * screen.
+	 * 
+	 * @return the GUI that was created.
+	 */
+	private Actor createLongTermDirection()
+	{
+		String communityOutreach[] = { "Community Outreach:",
+				"Building company's name recognition ",
+				"within the community through outreach ", "projects." };
+		String revenue[] = { "Revenue:", "Concrete, annual fiscal goals supporting ",
+				"revenue expectations." };
+		String customerService[] = { "Customer Service:",
+				"Maintain a long-lasting bond and ", "build a devoted following." };
+		
+		VerticalGroup longTermDirectionText = new VerticalGroup();
+		longTermDirectionText.addActor(createParagraph(communityOutreach));
+		longTermDirectionText.addActor(createParagraph(revenue));
+		longTermDirectionText.addActor(createParagraph(customerService));
+		
+		longTermDirectionText.space(25);
+		longTermDirectionText.left();
+		return longTermDirectionText;
+	}
+	
+	/**
+	 * This method creates the short term labels and button that will be displayed in the
+	 * screen.
+	 * 
+	 * @return the GUI that was created.
+	 */
+	private Actor createShortTermDirection()
+	{
+		String communityOutreach[] = { "Community Outreach:",
+				"Incentives for employees who volunteer",
+				"with designated community projects." };
+		String revenue[] = { "Revenue:", "One month contracts for advising",
+				"consultants" };
+		String customerService[] = { "Customer Service:",
+				"Host monthly drawings for free", "products or discounts on future ",
+				"purchases." };
+		
+		VerticalGroup longTermDirectionText = new VerticalGroup();
+		longTermDirectionText.addActor(createParagraph(communityOutreach));
+		longTermDirectionText.addActor(createParagraph(revenue));
+		longTermDirectionText.addActor(createParagraph(customerService));
+		
+		longTermDirectionText.space(25);
+		longTermDirectionText.left();
+		return longTermDirectionText;
+	}
+	
+	/**
+	 * A method that creates the paragraph that will be displayed and organize it in a
+	 * table.
+	 * 
+	 * @param text
+	 *            a string array of the text that will create the table.
+	 * @return The table created holding the text labels.
+	 */
+	private Table createParagraph(String[] text)
+	{
+		// REFACTOR: Combine with #createDifficultyChoiceDescriptionHard
+		Table requestedParagraph = new Table();
+		
+		List<Label> lines = new LinkedList<>();
+		// REFACTOR: Load lines from file
+		for (String stringLine : text)
+		{
+			lines.add(new Label(stringLine, skin));
+		}
+		
+		for (Label line : lines)
+		{
+			line.setAlignment(Align.left);
+			requestedParagraph.add(line).left();
+			requestedParagraph.row();
+		}
+		return requestedParagraph;
+	}
+	
+	/**
+	 * @return The Business Direction that was chosen by the user. "none" will be returned
+	 *         if no option was chosen.
+	 */
+	public String getChosenOption()
+	{
+		if (longTermOption.isChecked())
+			return "long term";
+		else if (shortTermOption.isChecked())
+			return "short term";
+		else
+			return "none";
+	}
```

Comment:
Flagged for refactoring: consider enumerating the return type.

Note: this can wait, but add a task on scrum and add a TODO line